packages/calcite-components/src/demos/list.html

```diff
@@ -576,6 +576,81 @@ <h1 style="margin: 0 auto; text-align: center">List</h1>
           </calcite-list>
         </div>
       </div>
+
+      <!-- simple list drag enabled -->
```

Comment:
I think we can drop the comments for each demo sections. This info is already displayed in each section's HTML.